transcom/mymove
# B19764 int 20240506 fe aoa packet change text

## Agility ticket
[B19764](https://www13.v1host.com/USTRANSCOM38/assetdetail.v1?number=B-19764)

## Summary

In the AOA Packet PDF download, the text label for shipment weight was changed from "- FINAL" to "- Estimated".

### How to test

* create a PPM move.  Select Advance Payment. 
* login as a SC
* find the move and open it
* Review the advance payment and approve it.
* Click the download link for the "AOA Packet PDF" and open the PDF.
* On page 1, there will be a shipment weight.  It should have the text "- Estimated"
* On page 2, there will be a shipment weight.  It should have the text "- Estimated"

## Screenshots
![image](https://github.com/transcom/mymove/assets/146897935/5b97e157-8a1a-4c25-9b9b-229c06b30d3b)

![image](https://github.com/transcom/mymove/assets/146897935/21c8019e-9b43-4860-8fbc-7299cd0071f0)
